src/vm/database/clarity_db.rs

```diff
@@ -455,3 +466,46 @@ impl <'a> ClarityDatabase <'a> {
         Ok(())
     }
 }
+
+// load/store STX token state and account nonces
+impl<'a> ClarityDatabase<'a> {
+    fn make_key_for_account(principal: &PrincipalData, data: StoreType) -> String {
+        format!("vm::{}::{}", principal, data as u8)
```

Comment:
This should probably have a different prefix from the `vm::` one used for general contract data to more explicitly avoid any keyspace overlap -- maybe `vm-account::{}::{}` ?

Reply:
Can do